shared/interop/generate_deposits_internal_test.go

```diff
@@ -0,0 +1,133 @@
+package interop
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/prysmaticlabs/prysm/shared/trieutil"
+)
+
+func TestSPMPDeposits(t *testing.T) {
```

Comment:
This test is really long, any way the setup can be split into smaller functions?

Reply:
I could break them out so that they were only testing one set of results at a time but that would result in a slowdown of the testing though as there would be a lot of recomputation (test 1 would generate the keys and confirm both methods gave the same results; test 2 would generate the keys then generate the deposit data and confirm both methods gave the same results, etc.).  Happy to leave it as-is or break them out; your call.

Reply:
Removed duplicated checks; test is smaller now.